scripts/create_patches_files.py

```diff
@@ -131,13 +131,27 @@ def make_api_request(url: str):
     patches = json.loads(r.text)
     return patches
 
-def get_patch_info(url: str, patch_id: Union[str, None] = None):
+def make_api_request_and_get_headers(url: str):
```

Comment:
Looking at this, it appears to be the same as `make_api_request` but also returns headers. Not fully sure why we need both. Might as well just have them always return headers as well imo

Reply:
`make_api_request` would return a tuple with `(headers, patches)` which would require manual unpacking with `[1]` to get the same result as before